code/espurna/light.ino

```diff
@@ -382,11 +384,10 @@ void _toRGB(char * rgb, size_t len) {
 
 void _toHSV(char * hsv, size_t len, bool target) {
     double h, s, v;
-    double brightness = (double) _light_brightness / LIGHT_MAX_BRIGHTNESS;
 
-    double r = (double) ((target ? _light_channel[0].target : _light_channel[0].inputValue) * brightness) / 255.0;
-    double g = (double) ((target ? _light_channel[1].target : _light_channel[1].inputValue) * brightness) / 255.0;
-    double b = (double) ((target ? _light_channel[2].target : _light_channel[2].inputValue) * brightness) / 255.0;
+    double r = static_cast<double>(target ? _light_channel[0].target : _light_channel[0].value);
+    double g = static_cast<double>(target ? _light_channel[1].target : _light_channel[1].value);
+    double b = static_cast<double>(target ? _light_channel[2].target : _light_channel[2].value);
```

Comment:
I noticed this strange calculation earlier. `inputValue` is `value` without brightness, so maybe it makes sense in that case. But it tries adjust it even with `target` which is set from `value`, thus displayed value could be wrong.
@CrazyIvan359 does checking for `target` flag makes any sense here at all? Web sends it with `false`, but mqtt reports with `true`... I am not quite sure what was the reason for this conditional.

Reply:
OK! Thanks

`bool target` param for `_toHSV(output, output_size, >> target <<)` function, selecting `channel.target` aka value+brightness via ternary 'if' above.

Last change to this was 19d0c5d17a576a22e8b7d38c1f09e3a14e41d6a5, which states to report target (adjusted) values to mqtt & web, but that only happened with mqtt (see `lightColor(false)` usage).
Maybe there's no need for conditional at all and it should always use adjusted value